File: DevelopArchitecture/DevelopArchitecture/architecture/baseView/JBaseCollectionViewCell.m
//
//  JBaseCollectionViewCell.m
//  DevelopArchitecture
//
//  Created by Jeffrey on 15/10/27.
//  Copyright © 2015年 Jeffrey. All rights reserved.
//

#import "JBaseCollectionViewCell.h"

@implementation JBaseCollectionViewCell
-(id)initWithFrame:(CGRect)frame{
    if(self=[super initWithFrame:frame]){
        self.backgroundColor = COLOR_BG_DARK;
        [self funj_addBaseCollectionView];
    }
    return self;
}
-(void)funj_addBaseCollectionView{
//    self.contentView addSubview:
}
-(void)funj_setBaseCollectionData:(NSDictionary*)data{
    
}
@end

File: DeveloperArchitecture/DeveloperArchitecture/architecture/baseView/JBaseCollectionViewCell.swift
//
//  JBaseCollectionViewCell.swift
//  DeveloperArchitecture
//
//  Created by Jeffrey on 2020/10/30.
//  Copyright © 2020 Jeffery. All rights reserved.
//

import Foundation
import UIKit

class JBaseCollectionViewCell: UICollectionViewCell {
    override init(frame: CGRect) {
        super.init(frame: frame)
        self.backgroundColor = kColor_Bg_Dark
        self.funj_addBaseCollectionView()
    }
    required init?(coder: NSCoder) {  fatalError("init(coder:) has not been implemented")}
    
    func funj_addBaseCollectionView(){
        
    }
    func funj_setBaseCollectionData(_ data : Any){
        
    }
}
